### ISSUE TITLE
Breaking Change: Get Object ACL

### ISSUE THREAD

**Issue by danielsf97** [minio/minio-go#1595]
I have recently upgraded the minio-go package from version v7.0.10 to v7.0.16 and the GetObjectACL method does not work anymore failing with error: "expected element type \<Owner\> but have \<Access Control Policy\>".
Anyone knows a solution to this problem?

Go version: 1.15

Thanks in advance.

**Comment by klauspost** [comment]
@danielsf97 You you add a bit more context to the error?

Which Go version are you using?

**Comment by harshavardhana** [comment]
> I have recently upgraded the minio-go package from version v7.0.10 to v7.0.16 and the GetObjectACL method does not work anymore failing with error: "expected element type but have ".

GetObjectACL implementation has not changed in years - closing this as there is no relevant reproducer present in the issue description.

**Comment by danielsf97** [comment]
> @danielsf97 You you add a bit more context to the error?
> 
> Also, which Go version are you using?

I'm using version 1.15.  Updated the issue.

**Comment by klauspost** [comment]
@vadmeste Could you check if this is a regression from #1538

**Comment by klauspost** [comment]
I think this makes it embedded:
```
type accessControlPolicy struct {
	Owner
	AccessControlList
}
```
Maybe it should be:

type accessControlPolicy struct {
	Owner Owner `xml:"Owner"`
	AccessControlList AccessControlList `xml:"AccessControlList"`
}

**Comment by vadmeste** [comment]
@klauspost I will check.

**Comment by minio-trusted** [comment]
This is fixed in latest release v7.0.18
